srcpkgs/fx/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'fx'
+pkgname=fx
+version=21.0.0
+revision=1
+build_style=go
+go_import_path="github.com/antonmedv/fx"
+hostmakedepends="git"
+short_desc="Terminal JSON viewer"
+maintainer="Gabriel M. Dutra <nulldutra@gmail.com>"
+license="MIT"
+homepage="https://github.com/antonmedv/fx"
+distfiles="$homepage/archive/$version.tar.gz"
```

Comment:
expand $homepage here

Reply:
In other packages, $homepage variable is not expanded. What motivation for this? but, I made the adjustment in the variable.

Reply:
having variables other than $version (like $homepage or $pkgname) in distfiles urls cause more changes to be needed if one changes. some packages did this before and they haven't been changed yet.